README.md

```diff
@@ -248,6 +261,31 @@ the `onLeave` and `onEnter` callback will be called. By using the arguments
 passed to the callbacks, you can determine whether the waypoint has crossed the
 top boundary or the bottom boundary.
 
+## Children
+
+If you don't pass children into your Waypoint, then you can think of the
+waypoint as a line across the page. Whenever that line crosses a
+[boundary](#offsets-and-boundaries), then the `onEnter` or `onLeave` callbacks
+will be called.
+
+If you do pass a child, it must be a single DOM Element (eg; a `<div>`).
+The `onEnter` callback will be called when *any* part of the child is visible
+in the viewport. The `onLeave` callback will be called when *all* of the child
+has exited the viewport (eg; both the top and bottom are either above or below).
+(Note that this is measured only on a single axis; strangely positioned elements
+may not work as expected).
+
+Deciding whether to pass children or not will depend on your use case. One
+example of when passing children is useful is for a scrollspy. Imagine if you
```

Comment:
One idea would just be to link to [the Boostrap docs](http://getbootstrap.com/javascript/#scrollspy). It might seem weird, but:

1. I hadn't really heard the term before them
2. they do a good job at explaining it
3. so many people use, or have used, Bootstrap

They version their docs, so we could "lock it down" to a specific version to avoid issues if they remove the scrollspy ever.

...maybe this is a bad idea.